src/controllers/HistoryController.ts

```diff
@@ -131,9 +131,15 @@ export class HistoryController extends RootComponent {
     return diff;
   }
 
-  private getHashValue(value: string): any {
-    Assert.isNonEmptyString(value);
-    return HashUtils.getValue(value, HashUtils.getHash(this.windoh));
+  private getHashValue(key: string): any {
+    Assert.isNonEmptyString(key);
+    let value;
+    try {
+      value = HashUtils.getValue(key, HashUtils.getHash(this.windoh));
+    } catch (error) {
+      this.logger.error(`Could not parse parameter ${key} from URI`);
```

Comment:
What happens in the `HistoryController` class when this function return `undefined` ? Does it keep going correctly ?

My question is : if we have `#q=something_legit&f:@facet={something_not_legit}&first=10` , what we want is 

* q to be set correctly (and is rendered correctly in the search box)
* facet is in error, and nothing happens for this particular facet
* the pager renders the first page

Is it the case ?

If it is, I would like an explicit test in the HistoryController class to ensure that it stays like this.